README.md

```diff
@@ -6,76 +6,51 @@
 [![GitHub release](https://img.shields.io/github/release/kubernetes/dashboard.svg)](https://github.com/kubernetes/dashboard/releases/latest)
 [![Greenkeeper badge](https://badges.greenkeeper.io/kubernetes/dashboard.svg)](https://greenkeeper.io/)
 
-Kubernetes Dashboard is a general purpose, web-based UI for Kubernetes clusters. It allows users to
-manage applications running in the cluster and troubleshoot them, as well as manage the cluster
-itself.
+Kubernetes Dashboard is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications
+running in the cluster and troubleshoot them, as well as manage the cluster itself.
 
 ![Dashboard UI workloads page](docs/dashboard-ui.png)
 
-## Deployment
-It is likely that the Dashboard is already installed on your cluster. Check with the following command:
-```shell
-$ kubectl get pods --all-namespaces | grep dashboard
-```
+## Getting Started
 
-If it is missing, you can install the latest stable release by running the following command:
-```shell
-$ kubectl create -f https://git.io/kube-dashboard
-```
+**IMPORTANT:** Since version 1.7 Dashboard uses more secure setup. It means, that by default it has minimal set of
+privileges and can only be accessed over HTTPS. It is recommended to read [Access Control](
+https://github.com/kubernetes/dashboard/wiki/Access-control) guide before performing any further steps.
 
-If you are using Dashboard on a cluster with disabled authorization, you can install the latest stable release by running the following command:
-```shell
-$ kubectl create -f https://git.io/kube-dashboard-no-rbac
-```
+To deploy Dashboard execute following command:
 
-You can also install unstable HEAD builds with the newest features that the team works on by
-following the [development guide](docs/devel/head-releases.md).
+```sh
+$ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
```

Comment:
what about the git.io shorthand? (BTW: I think there might be some management problem, because edit rights belong to users and cannot be assigned to the project - I think)

Reply:
We will use it but first we need to actually merge it and make sure that shortened link won't change.

Reply:
> BTW: I think there might be some management problem, because edit rights belong to users and cannot be assigned to the project - I think)

What do you mean?

Reply:
Sorry, forget about it...